src/kit/Glossary.module.scss

```diff
@@ -11,20 +11,33 @@
     .label {
       align-self: flex-start;
       flex: 0 0 34%;
-      font-size: 12px;
```

Comment:
the valueList has a set font-size, do you want one for the label or no?

Reply:
Good catch, we can remove both of those since I added `Label` and `Body`